Xamarin.Forms.Platform.iOS/Renderers/NavigationRenderer.cs

```diff
@@ -676,6 +677,10 @@ void UpdateBarBackgroundColor()
 					navigationBarAppearance.BackgroundColor = barBackgroundColor.ToUIColor();
 				}
 
+				var barBackgroundBrush = NavPage.BarBackground;
```

Comment:
Does this code need to be changed so it only executes if the background brush is actually set?